pkg/gds/config/config.go

```diff
@@ -41,6 +41,12 @@ type AdminConfig struct {
 	Enabled  bool   `split_words:"true" default:"true"`
 	BindAddr string `split_words:"true" default:":4434"`
 	Mode     string `split_words:"true" default:"release"`
+
+	// TokenKeys are the paths to RSA JWT signing keys in PEM encoded format. The
+	// environment variable should be a comma separated list of keyid:path/to/key.pem
+	// Multiple keys are used in order to rotate keys regularly; keyids therefore must
+	// be sortable; in general we prefer to use ksuid for key ids.
+	TokenKeys map[string]string `split_words:"true" required:"true"`
```

Comment:
Really quite happy that envconfig handles extracting maps from environment variables, very cool!

Reply:
This is neat! 

So the picture I have in my head is that keys live in a file called `ksuid.pem` (or similar), to which we append a new key every 3 months. 

`envconfig` (routinely? continuously? on reboot?) looks in that file and computes the 2 (?) most recent keys; the newest one, which we use for client token generation and verification, and the second-to-newest one, which we can use for client token verification.

do I have that right?